.gitignore

```diff
@@ -1 +1,2 @@
 __pycache__
+hangbot.log
```

Comment:
Change this to `hangbot.log*`
https://docs.python.org/2/library/logging.handlers.html#rotatingfilehandler